blueocean-personalization/package.json

```diff
@@ -3,19 +3,19 @@
   "version": "0.0.1",
   "scripts": {
     "storybook": "start-storybook -p 9001 -s node_modules/@jenkins-cd/design-language/dist/assets/,target/classes/org/jenkins/ui/jsmodules/blueocean_personalization/",
-    "lint": "gulp lint",
-    "lint:fix": "gulp lint --fixLint",
-    "lint:watch": "gulp lint:watch --continueOnLint",
-    "test": "gulp test",
-    "test:watch": "gulp test:watch",
-    "bundle": "gulp bundle",
-    "bundle:watch": "gulp bundle:watch",
-    "mvnbuild": "gulp bundle",
-    "mvntest": "gulp test lint"
+    "lint": "jjsbuilder --tasks lint",
```

Comment:
I don't think so. Anyone that has worked in node land for more than a day knows they need to check the `scripts` in the `package.json`.